Description: Excludes the Avalon Logger support (not in Debian and obsolete)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -205,6 +205,7 @@
       classpathref="ivy.dep"
       bootclasspath="${boot.classpath.j2se1.4}"
       excludes="
+        freemarker/log/_AvalonLoggerFactory.java,
         freemarker/ext/beans/_EnumModels.java,
         freemarker/ext/beans/JRebelClassChangeNotifier.java,
         freemarker/ext/dom/SunInternalXalanXPathSupport.java,
